account/src/main/java/com/user/account/controller/AllControllers.java

```diff
@@ -0,0 +1,71 @@
+package com.user.account.controller;
+import java.util.List;
+import com.user.account.advice.CustomExceptions;
+import com.user.account.entity.FundTransferDetails;
+import com.user.account.message.DefaultMessage;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.HttpStatus;
+import org.springframework.http.ResponseEntity;
+import org.springframework.validation.annotation.Validated;
+import org.springframework.web.bind.annotation.*;
+import com.user.account.entity.User;
+import com.user.account.entity.Transactions;
+import com.user.account.services.AccountService;
+import com.user.account.services.TransactionService;
+import javax.validation.constraints.Min;
+import static com.user.account.dataValidation.TransactionUsersAndAmountValidation.transactionUsersAndAmountValidation;
+
+@Validated
+@RestController
+@RequestMapping("/")
+public class AllControllers {
+
+    @Autowired
+    private AccountService accountService;
+
+    @Autowired
+    private TransactionService transactionService;
+
+    //Send the response to the user based on transaction status
+    public ResponseEntity<String> transactionResponse(boolean transactionStatus){
+        if(transactionStatus)
+            return new ResponseEntity<>(DefaultMessage.TRANSACTION_SUCCESS, HttpStatus.OK);
+        else
+            return new ResponseEntity<>(DefaultMessage.TRANSACTION_FAILED_LOW_BALANCE, HttpStatus.BAD_REQUEST);
+    }
+
+    // Updating accountBalance after debit
```

Comment:
Don't put comments like this, prefer Javadoc

Reply:
Added JavaDoc documenattion